xml/admin_administration.xml

```diff
@@ -51,17 +51,19 @@
   </itemizedlist>
 
   <para>
-   Transactional updates are managed by the
-   <command>transactional-update</command> script, which is called once a day.
-   The script checks if any updates are available. If there are any updates to
-   be applied, a new snapshot of the root file system is created and the system
-   is updated using the <command>zypper dup</command>. All updates released to
-   this point are applied. The snapshot is then marked as active and will be
-   used after the next reboot of the system. Ensure that the cluster is
-   rebooted as soon as possible after the update installation is complete,
-   otherwise all changes will be lost.
+   Transactional updates are managed by the <command>transactional-update</command> 
```

Comment:
I would add a paragraph here:

Updating is a two step process. First transactional update is run on individual machines to apply the updates for the next boot. Second, the update  needs to be applied. The first step is run automatically every day, the second step is orchestrated from Velum so that any configuration updates can be done as part of the update.

Reply:
I've just rewritten a section explaining this.